fix build failed, update version to master of finatra consistent

lujian0571 opened the pull request:
scrooge-sbt-plugin=4.18.0
scala=2.12.1
sbt.version=0.13.15

Commit:
fix build failed, update version to master of finatra consistent
scrooge-sbt-plugin=4.18.0
scala=2.12.1
sbt.version=0.13.15

jcrossley commented on build.sbt:
```diff
@@ -15,7 +15,7 @@ lazy val versions = new {
 lazy val baseSettings = Seq(
   version := "1.0.0-SNAPSHOT",
-  scalaVersion := "2.11.8",
+  scalaVersion := "2.12.1",
```
Instead of only building for 2.12, let's cross-build for both 2.11 and 2.12. As a reference, we do that in Finatra here: https://github.com/twitter/finatra/blob/develop/build.sbt

Commit:
scalaVersion := "2.11.11",
crossScalaVersions := Seq("2.11.11", "2.12.1")

Commit:
scroogeThriftDependencies for crossScalaVersions

if(scalaVersion.value.startsWith("2.11")) "finatra-thrift_2.11" else "finatra-thrift_2.12"

lujian0571 commented:
@jcrossley Changed.
 crossScalaVersions: 2.11.11 and 2.12.1

jcrossley reviewed (approved):
Looks good, thanks for the contribution! I'm going to pull this in internally and I'll let you know when it's merged!

jcrossley commented:
Merged! Thanks for the PR!
